src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -107,12 +99,12 @@ default String checkUrlEquals(String expectedURL, double waitFor, double timeToo
      * @param timeTook      - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: the actual title
      */
-    default String checkTitleEquals(String expectedTitle, double waitFor, double timeTook) {
-        String actualTitle = getApp().get().title();
+    String checkTitleEquals(String expectedTitle, double waitFor, double timeTook) {
+        String actualTitle = this.app.get().title();
         if (!actualTitle.equals(expectedTitle)) {
-            getReporter().fail("to be on page with the title of <b>" + expectedTitle + "</b>", waitFor, "The page title reads <b>" + actualTitle + "</b>", timeTook);
+            this.reporter.fail("to be on page with the title of <b>" + expectedTitle + "</b>", waitFor, "The page title reads <b>" + actualTitle + "</b>", timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "The page title reads <b>" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)